src/compressAlignment.cpp

```diff
@@ -57,10 +70,145 @@ void shasta::compress(const Alignment& alignment, string& s)
 }
 
 
+void shasta::decompress(const string& s, Alignment& alignment)
+{
+    uint32_t ordinal0 = 0;
+    uint32_t ordinal1 = 0;
+    
+    size_t pos = 0;
+    while (pos < s.size()) {
+        uint32_t streakSize;
+        int32_t skip0, skip1;
+
+        uint8_t fid = shasta::compressAlignment::extractFormatIdentifier(s[pos]);
```

Comment:
I prefer long names (`formatIdentifier` instead of `fid`) but this is subjective and not a requirement.